server/sonar-server/src/test/java/org/sonar/server/ws/WebServiceEngineTest.java

```diff
@@ -347,8 +347,9 @@ public void define(Context context) {
         .setHandler((request, response) -> {
           throw BadRequestException.create("Bad request !");
         });
-      createNewDefaultAction(newController, "fail_with_multiple_messages")
-        .createParam("count", "Number of error messages to generate")
+      NewAction failWithMultipleMessages = createNewDefaultAction(newController, "fail_with_multiple_messages");
+      failWithMultipleMessages.createParam("count").setDescription("Number of error messages to generate");
```

Comment:
This is not possible, because the "new" `createParam` method returns `NewParam`, whereas the old implementation returned `NewAction`.

Btw: Scnr: This api should get refactored some day, this will allow inlining everywhere... :)